[JA] Update First-Party Sets translation

ilyaspiridonov opened the pull request:
Updates Japanese translation of privacy-sandbox/First-Party Sets 
CC: @agektmr

agektmr commented on site/ja/docs/privacy-sandbox/first-party-sets/index.md:
```diff
@@ -1,24 +1,26 @@
-subhead: >
+layout: layouts/doc-post.njk
+title: First-Party Sets
+subhead: |2
```
Can you fix these kind of glicthes?

ilyaspiridonov replied:
done!

agektmr commented on site/ja/docs/privacy-sandbox/first-party-sets/index.md:
```diff
@@ -67,19 +69,17 @@ First-Party Sets を補完する提案は、`SameParty` Cookie 属性です。 C
+- [First-Party Sets 属性と sameParty 属性](/blog/first-party-sets-sameparty/)
+- [First-Party Sets の技術 Explainer](https://github.com/privacycg/first-party-sets)
+- [Chrome プラットフォームのステータス](https://chromestatus.com/feature/5640066519007232)
+- [Chromium プロジェクト](https://www.chromium.org/updates/first-party-sets)
```
I didn't change any of lines above which turns `*` into `-`. LIne 76-78 and 82-85. Are these done by GitLocalize?

ilyaspiridonov replied:
Correct, by GitLocalize. Both `*` and `-` are the correct notations for markdown bullet lists, and GL's reverse parser uses the `-` notation. 
We figured this shouldn't be a problem as long as 11ty renders correct html from this (which it does): 
<img width="580" alt="Screenshot 2022-08-26 at 15 59 11" src="https://user-images.githubusercontent.com/8548803/186909078-9a4cfb9b-a5cf-4018-8bda-13b87aa4d09d.png">

If there's another reason this is important, please let me know and we'll look into this with a dev team (or set up a GitHub action on our fork to correct this automatically).

agektmr replied:
I agree this shouldn't be a problem, but I'm curious why this conversion occurred if GitLocalize only translates updated fragments. Also, I didn't touch the section at all. When did this conversion occur?

agektmr replied:
I see. Let's revisit when this causes any problems.

agektmr reviewed (approved):
LGTM

agektmr commented:
Holding off on merging this until confirming my last comment is delivered to @ilyaspiridonov

agektmr commented:
@ilyaspiridonov looks like it's complaining about CLA. Do you have any solutions for it?

ilyaspiridonov commented:
@agektmr I have rebased against upstream, which removed commits authored by our fork sync bot. So this PR is now green. 
We have updated the GitHub action on our fork, so all future commits that sync our fork with upstream should be authored by a user that has a signed CLA. 
Thank you!
